backend/src/main/java/co/inventorsoft/scripty/model/dto/MockRequestDto.java

```diff
@@ -0,0 +1,47 @@
+package co.inventorsoft.scripty.model.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Pattern;
+import java.util.Map;
+
+/**
+ * @author A1lexen
+ */
+
+@Getter
+@Setter
+@ApiModel(value="Mock Request")
+public class MockRequestDto {
+    @ApiModelProperty(value = "Returned status. Default value '200'.")
+    private int status = 200;
```

Comment:
Do not use primitive types for nullable things. Please, use `Integer` wrapper + `@NotNull` contraint instead